packages/gitbook/src/lib/visitors.ts

```diff
@@ -82,6 +121,106 @@ export function getVisitorToken({
     }
 }
 
+/**
+ * Get the visitor unsigned/public claims for the request. They can either be in `visitor.` query
```

Comment:
Should it be `gitbook.` instead of `visitor.`?

Reply:
We should reserve `gitbook.` for something else. Let's keep `visitor.` for clarity